Description: Strip versions from links to jQuery scripts
 jQuery libraries bundled with uwsgicc are not packaged. Instead, symbolic
 links to jQuery libraries from libjs-jquery* packages are created.
 .
 jQuery libraries from libjs-jquery* packages could be newer (or older) than
 jQuery libraries bundled with uwsgicc. Unmatched version in name of script
 could be confusing.
Author: Leonid Borisenko <leo.borisenko@gmail.com>
Forwarded: not-needed
Last-Update: 2011-05-26

Index: uwsgi/uwsgicc/templates/index.html
===================================================================
--- uwsgi.orig/uwsgicc/templates/index.html	2011-05-26 11:36:25.000000000 +0300
+++ uwsgi/uwsgicc/templates/index.html	2011-05-26 11:36:50.000000000 +0300
@@ -5,8 +5,8 @@
 <title>uWSGI {{uwsgi.version}} Control Center</title>
 
 <link href="{{ url_for('static', filename='css/custom-theme/jquery-ui-1.8.14.custom.css') }}" rel="stylesheet" type="text/css"/>
-<script src="{{ url_for('static', filename='js/jquery-1.5.1.min.js') }}"></script>
-<script src="{{ url_for('static', filename='js/jquery-ui-1.8.14.custom.min.js') }}"></script>
+<script src="{{ url_for('static', filename='js/jquery.min.js') }}"></script>
+<script src="{{ url_for('static', filename='js/jquery-ui.min.js') }}"></script>
 
 <style type="text/css"> 
 body {
